gradle.properties

```diff
@@ -0,0 +1,6 @@
+//signing.keyId=<publicKey>
```

Comment:
The idea was to leave this only as a template to show what needs to be done. I.e. As it is now and to never check in with real credentials. The credential values won't be accessed unless someone tries to publish anyway I believe. 

But we can remove this file if you think that is better.